README.md

```diff
@@ -118,6 +124,12 @@ Parameters:
 - `role`: Name of the role, e.g: `caseworker-divorce`.
 - `classification`: Optional. One of `PUBLIC`, `PRIVATE` or `RESTRICTED`. Defaults to `PUBLIC`.
 
+#### Note:
```

Comment:
could we please remove this note section? jq is already listed in the required tools list and a link to its own installation page is provided.

Reply:
done